### DISCUSSION TITLE
Approved for Student Developer and still havent received confirmation email and benefits

### DISCUSSION THREAD

**Discussion by Khatriprakhar1** [community/community#140914] [GitHub Education]
### Hi everyone,

Approved for Student Developer and still havent received confirmation email and benefits kindly consider it and resolve this Thanks
![githubDev](https://github.com/user-attachments/assets/bf91979d-80b1-4e0e-a200-9e01d1d94cf3)

**Comment by davevad93** [comment]
Hi @Khatriprakhar1 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Khatriprakhar1 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Khatriprakhar1** [reply]
its been a while and stil havent got my benefits ?

**Reply by davevad93** [reply]
Don't worry @Khatriprakhar1 , you have been approved just 3 days ago, as I said before it's normal to wait even a couple of weeks when there's a high volume of requests.

**Reply by Khatriprakhar1** [reply]
alright would give it a go and wait i am just so excited for it i just cant wait lmao !thanks

**Reply by davevad93** [reply]
You're welcome @Khatriprakhar1 👍
